.changeset/unlucky-oranges-notice.md

```diff
@@ -0,0 +1,6 @@
+---
+'@twilio-paste/sibling-box': patch
+'@twilio-paste/core': patch
+---
+
+[Sibling Box] Remove obsolete eslint ignore statement; cosmetic change only.
```

Comment:
suggestion: same as for Modal, we can just make this change for us and not ship a new version to users as nothing changes in the compiled code